### PR TITLE
fix #926

### LINKED ISSUE #926: Symbol 2BAD doesn't render
The symbol in http://unicode-table.com/en/2BAD/ doesn't render properly in many OSes by default such as Windows 10 and Mac OSX.

**Solution:-**
Replace that symbol from the following files

- [**camperbot.md** (wiki)](https://github.com/FreeCodeCamp/wiki/blob/master/camperbot.md)
- [**Help-Rooms.md** (wiki)](https://github.com/FreeCodeCamp/wiki/blob/master/Help-Rooms.md)

with the symbol in http://unicode-table.com/en/2191/
&nbsp;
<br>
<br>

> ----
<p align="center">Need Help? Read <a href="https://github.com/FreeCodeCamp/wiki/blob/master/.github/CONTRIBUTING.md"><b>CONTRIBUTING</b></a> Guidelines<br>or Chat with us in <kbd><a href="https://gitter.im/FreeCodeCamp/Wiki"> <b>FreeCodeCamp/Wiki</b></a></kbd></p>

> ----

**Currently Assigned to:** none

> *If you want to work on this, please comment below or we can chat over* <kbd><a href="https://gitter.im/FreeCodeCamp/Wiki"> <b>gitter</b></a></kbd>
Note: you may be assigned **only if** the current assignee doesn't respond within the given time-frame of 7 days.

### PR DESCRIPTION
**Pull request by akarshHegde**

### CODE REVIEW

**Comment by theflametrooper** [comment]
### **LGTM :+1:**

Would you mind fixing https://github.com/FreeCodeCamp/wiki/blob/master/camperbot.md as well?

**Comment by akarshHegde** [comment]
Sure, already working on it!

**Comment by akarshHegde** [comment]
Done :)

**Comment by theflametrooper** [comment]
#### :sparkles: :sparkles: **LGTM :+1:** :sparkles: :sparkles: 

:tada: Thanks for contributing @akarshHegde and congratulations on your first PR submission to FreeCodeCamp's Wiki! :tada:

**Comment by akarshHegde** [comment]
Thanks @theflametrooper . That did feel good! :)

File: Help-Rooms.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ _Choose the right room for your questions!_
 
 See also: :earth_asia: [**Language Specific**](#language-specific-rooms-) **|** :satellite: [**Technology**](#technology-based-rooms-) **|** :crystal_ball: [Miscellaneous](#miscellaneous-rooms-)
 
-[**`back to top`⮭**](#freecodecamp-chat-rooms)
+[**`back to top`&#8593;*](#freecodecamp-chat-rooms)
```
Comment (abhisekp) [inline Help-Rooms.md]: Please use the symbol and add `**` after the symbol. Currently, the Markdown is invalid.

Comment (theflametrooper) [inline Help-Rooms.md]: @abhisekp Addressed in #972

**Pull request merged**